migrations/V121__Add_TRB_Request_Feedback_Table.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TYPE trb_feedback_action AS ENUM (
+	'REQUEST_EDITS',
+	'READY_FOR_CONSULT'
+);
+
+CREATE FUNCTION check_eua_array_regex (
+    arr TEXT[], regex TEXT
+) RETURNS BOOLEAN AS $$
+
+    SELECT bool_and (n ~ regex)
+    FROM unnest(arr) s(n);
```

Comment:
Same question out of curiosity as before - what does `s(n)` do here? From experimenting locally, it looks like it names the column returned from `unnest` as `n`, is that the idea?

Reply:
Still curious about this, if you don't mind answering.

Reply:
[See this](https://stackoverflow.com/questions/38718553/set-a-check-constraint-for-text-array-in-postgresql)

Reply:
Makes sense for now, and it works.